features/app/constants.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Available application screens/routes.
+ * 
+ * @enum {string}
+ */
+export const APP_SCREEN = {
+    WELCOME: 'welcome',
+    CONFERENCE: 'conference'
```

Comment:
Please comply with the coding style which requires sorting by alphabetical order!